publish-site.sh

```diff
@@ -42,6 +42,12 @@ if [[ "$1" == "--docker" ]]; then
   docker run -ti --name=vitess_publish_site -v $PWD/vitess.io:/in vitess/publish-site bash -c \
     'cp -R /in /out && cd /out && bundle install && bundle exec jekyll build'
   docker cp vitess_publish_site:/out/. vitess.io/
+  # There are cases where docker cp copies the contents of "out" in a
```

Comment:
I wasn't clear enough in my comment: In my case it always failed.

I assume it didn't fail before for other people, probably at a different docker version?

re: fixing the copy command:
I played around with the docker cp command and tried to get the old behavior where it copies the content directly. I tried, but I didn't succeed :)